content/concepts/interoperability/cross-chain-bridges.md

```diff
@@ -0,0 +1,107 @@
+---
+html: cross-chain-bridges.html
+parent: xrpl-interoperability.html
+blurb: Cross-chain bridges for the XRP Ledger enable value in the form of XRP and other tokens (IOUs) to move efficiently between blockchains.
+labels:
+  - Blockchain
+---
+# Cross-Chain Bridges
+
+Cross-chain bridges for the XRP Ledger enable value in the form of XRP and other tokens (IOUs) to move efficiently between blockchains such as the XRP Ledger and its sidechains.
+
+
+A locking chain is a blockchain that holds assets that are then put into trust when a bridge to an issuing chain is created.
+
+An issuing chain is an independent ledger with its own consensus algorithm and transaction types and rules. It acts as its own blockchain.
+
+Both the locking and issuing chains operate as parllel networks with independent nodes and validators. They rely on independent [witness servers](witness-server.html) to watch transactions between the two chains and attest that assets have moved into specifically designated accounts.
+
+## Terminology
+
+* Bridge: A method of moving assets/value from one blockchain to another.
+
+* Witness: Independent servers that are aware of the locking and issuing chains. See [witness servers](witness-server.md) for more information.
+
+* Cross-chain transfer: A transfer of assets from one chain to another.
+
+* Source chain: The chain that a cross-chain transfer begins from. The transfer is from the source chain and to the destination chain.
+
+* Destination chain: The chain that a cross-chain transfer ends at. The transfer is from the source chain and to the destination chain.
+
+* Locking chain: The chain on which the bridge locks and unlocks assets.
+
+* Issuing chain: The chain on which the bridge mints and burns assets.
+
+* Door account: A special type of account that is used to move assets from one chain to another. The door account on a locking chain is used to put assets into trust, and the door account on an issuing chain used to issue wrapped assets. 
+
+* Cross-chain claim ID: A special identifier used for cross-chain transfers. A cross-chain claim ID represents *one* cross-chain transfer of value.
+
+## How Do Cross-Chain Transactions Work?
+
+### Pre-requisites
+
+You must ensure that the following are set up and running before initiating cross-chain transactions. 
+
+* Ensure that the witness server(s) are up and running.
+* Set up a bridge between the two chains, including a _door account_ on each chain. On one chain, the asset is locked and unlocked, hence the name "locking chain", and on the other chain, assets are minted and burned, or issued and reclaimed, hence the name "issuing chain".
+* When using the XRP Ledger Mainnet as one of the chains, ensure that  the `Sidechains` amendment is enabled in the `rippled.cfg` configuration file. 
+
+### Working of Cross-Chain Transactions 
+ 
+At a high-level, cross-chain transactions involve the following steps: 
```

Comment:
This flow is good, but perhaps there should also be pointers to the actual transactions submitted.